### PR TITLE
(DOCSP-26883) Fixes rST formatting error that broke See Also notes

### PR DESCRIPTION
**Pull request by sarahsimpers**
[Jira ticket](https://jira.mongodb.org/browse/DOCSP-26883)

### CODE REVIEW

**Review by corryroot — APPROVED** [review]
LGTM!

File: source/style-guide/style/text-formatting.txt [hunk 1]
```diff
@@ -43,7 +43,9 @@ elements. The following style differences are highlighted:
    Application names
      Apply no additional formatting.
 
-     .. seealso:: :ref:`product-names-version-numbers`
+     .. seealso:: 
+       
+        :ref:`product-names-version-numbers`
```
Comment (ccho-mongodb) [inline source/style-guide/style/text-formatting.txt:48]: Just to match the other entries:

```suggestion
        :ref:`product-names-version-numbers`.
```

**Review by ccho-mongodb — APPROVED** [review]
LGTM

File: source/style-guide/style/text-formatting.txt (hunk 1 above)
Comment (ccho-mongodb) [reply source/style-guide/style/text-formatting.txt:48]: @sarahsimpers could you make the above change when you get a chance, thanks!

Comment (ccho-mongodb) [reply source/style-guide/style/text-formatting.txt]: Ah sorry, I didn't realize you were waiting. I'll ask the other reviewers to take a look as soon as they get the chance.

Per our style guide team rep process, we'll merge and deploy once everything is LGTM-ed.

**Review by jeff-allen-mongo — APPROVED** [review]
LGTM, thanks @sarahsimpers !

**Review by krollins-mdb — APPROVED** [review]
Apologies for my slower review. LGTM!

**Pull request merged**